pkgs/games/devilutionx/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake, SDL2, SDL2_mixer, SDL2_ttf, libsodium, pkg-config }:
+stdenv.mkDerivation rec {
+  version = "unstable-2019-07-28";
+  pname = "devilutionx";
+
+  src = fetchFromGitHub {
+    owner = "diasurgical";
+    repo = "devilutionX";
+    rev = "b2f358874705598ec139f290b21e340c73d250f6";
+    sha256 = "0s812km118qq5pzlzvzfndvag0mp6yzvm69ykc97frdiq608zw4f";
+  };
+
+  NIX_CFLAGS_COMPILE = "-I${SDL2_ttf}/include/SDL2";
+
+  # compilation will fail due to -Werror=format-security
+  hardeningDisable = [ "format" ];
+
+  nativeBuildInputs = [ pkg-config cmake ];
+  buildInputs = [ libsodium SDL2 SDL2_mixer SDL2_ttf ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp devilutionx $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/diasurgical/devilutionX";
+    description = "Diablo build for modern operating systems";
+    license = licenses.unlicense;
+    maintainers = [ maintainers.karolchmist ];
+    platforms = platforms.linux;
```

Comment:
I'm guessing getting this to run on the other platforms supported might not be trivial and you don't have the software to do so?

Reply:
Yes, but maybe we should put `platforms.all` and see if it compiles ? In theory it should be possible to compile on MacOS and Windows as well.

(I have only Linux to my disposition)

Reply:
Sure, give it a try and we can get the bot to give it a try.